src/libraries/System.Private.CoreLib/src/System/Diagnostics/Tracing/EventSource.cs

```diff
@@ -3840,11 +3840,78 @@ internal static void InitializeDefaultEventSources()
             {
                 const string name = "System.Diagnostics.Metrics";
                 Guid id = new Guid("20752bc4-c151-50f5-f27b-df92d8af5a61");
-                PreregisterEventProviders(id, name, GetMetricsEventSource);
+                EventSourceInitHelper.PreregisterEventProviders(id, name, EventSourceInitHelper.GetMetricsEventSource);
             }
         }
 
-        private static EventSource? GetMetricsEventSource()
+        // private instance state
```

Comment:
Nit: It would be nice to move the fields to the top to follow the coding style (https://github.com/dotnet/runtime/blob/main/docs/coding-guidelines/coding-style.md: Fields should be specified at the top within type declarations.)

Reply:
done 👍